spec/datadog/core/telemetry/heartbeat_spec.rb

```diff
@@ -6,7 +6,7 @@
   subject(:heartbeat) { described_class.new(enabled: enabled, interval: interval, &block) }
 
   let(:enabled) { true }
-  let(:interval) { 60 }
+  let(:interval) { 1.2 }
```

Comment:
Minor: I suggest propagating the `_seconds` prefix internally as well:

```suggestion
  let(:heartbeat_interval_seconds) { 1.2 }
```